dbtmetabase/parsers/dbt_folder.py

```diff
@@ -29,12 +29,12 @@ def __init__(self, project_path: str):
     def read_models(
         self,
         database: str,
-        schema: str,
+        schema: Optional[str] = None,
         schema_excludes: Iterable = None,
         includes: Iterable = None,
         excludes: Iterable = None,
         include_tags: bool = True,
-        docs_url: str = None,
+        dbt_docs_url: str = None,
```

Comment:
Prefix `dbt_` makes sense for entry point options, but here it's going into dbt-only context, so I think `docs_url` is enough.

Reply:
agreed, adjusted accordingly